### PR TITLE
[fix] Quick fix for backup_core_only feature with the old backup.

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *https://forum.yunohost.org/t/nextcloud-failed-update-and-restore/7565/9*
- *Fix https://github.com/YunoHost-Apps/nextcloud_ynh/issues/185*

## Solution
- *Keep `backup_core_only` for compatibility with the previous backup script.*

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [x] **Approval (LGTM)** : Kay0u
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/nextcloud_ynh%20PR186/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
As we currently have a testing in progress.
I suggest to merge that fix directly in master and fix immediately that issue.

File: scripts/upgrade [hunk 1]
```diff
@@ -56,8 +56,16 @@ current_major_version=${current_version%%.*}
 
 if [ $current_major_version -gt 11 ]
 then
+    # Inform the backup/restore process that it should not save the data directory
+    # Use only for the previous backup script that doesn't set 'is_big'
+    ynh_app_setting_set $app backup_core_only 1
```
Comment (kay0u) [inline scripts/upgrade]: `if [ $current_major_version -lt 15 ]` maybe?

Comment (maniackcrudelis) [reply scripts/upgrade]: This variable will just be ignored if the backup script is recent

**Comment by JimboJoe** [comment]
I'm OK to merge this quickly into master. Shouldn't we change the version number...?

**Comment by maniackcrudelis** [comment]
Do we agree to merge quickly (this evening ?) into master ?

**Comment by maniackcrudelis** [comment]
Indeed we should !
I always forget that...

**Pull request merged**